pkg/plugins/golang/v2/api.go

```diff
@@ -207,6 +207,11 @@ func (p *createAPISubcommand) PostScaffold() error {
 		return fmt.Errorf("unknown pattern %q", p.pattern)
 	}
 
+	err := util.RunCmd("Running go mod tidy", "go", "mod", "tidy")
```

Comment:
Let's keep the message descriptive, as the exact command run will already be logged: https://github.com/kubernetes-sigs/kubebuilder/pull/2037/checks?check_run_id=1980219260#step:5:67

```suggestion
	err := util.RunCmd("Update dependencies", "go", "mod", "tidy")
```
Same for v3. Also, could you also update this message for init v2 and v3 subcommands for consistency?

Reply:
I updated the output messages for both init and api subcommand for both v2 and v3.